src/python/constants.py

```diff
@@ -17,10 +15,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import multiprocessing
```

Comment:
are we using multiprocessing?

Reply:
It's used to get cpu count [here](https://github.com/mikemccand/luceneutil/blob/main/src/python/constants.py#L115). I reordered the imports so it showed up in the diff.